user/libtermkey/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "libtermkey"
```

Comment:
we previously removed libtermkey because it's dead/deprecated upstream

Reply:
unfortunately it is still a dependency of vis.

https://github.com/martanne/vis/issues/1071

Reply:
what do you think about vendoring libtermkey within user/vis? 

> Besides it consists of only 3 C files which could always be included in vis directly (but I prefer not to do so).